rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp-submit-document-to-mulesoft-idp.adoc

```diff
@@ -0,0 +1,60 @@
+= Submit Document to MuleSoft IDP 
+
+Send documents to MuleSoft IDP to process them with your published document actions. Each execution generates an Execution ID that you must use when you run the Retrieve Results from MuleSoft IDP action step to query the results of said execution. 
+
+Calls to IDP from RPA Builder consume automation credits per page processed in addition to the bot minutes consumed by the RPA process. See xref:rpa-home::ms-automation-credits-usage-types.adoc[] for conversion rates and usage details. 
+
+== Before You Begin
+
+Before you begin using this action step, ensure that: 
+
+* Your Anypoint user account has the *Build Actions* permission
+* You have IDP document actions published in your organization
++
+See xref:idp::publishing-document-actions.adoc[] for additional information. 
+* You have a Connected App configured to access IDP
++
+See xref:idp::automate-document-processing-with-rpa.adoc[] for configuration instructions. 
+
+== Properties
+
+This action step has no properties other than *Common* and *Timing*. See xref:common-properties.adoc[] for further details. 
+
+== Inbound Variables
+
+* *Directory Path*
++
+Path to the directory where the file to submit is located.
+* *File Name*
++
+Name of the file to submit.
+* *Client ID*
++
+Client ID of the Connected App configured to call IDP.
+* *Client Secret*
++
+Client Secret of the Connected App configured to call IDP.
+* *Connected App Credentials*
++
+Pin credentials containing the Client ID and Client Secret of the Connected App configured to call IDP.
+* *Execution ID*
++
+Execution ID of the document processing operation. 
+
+== Outbound Variables
+
+* *Success*
+* *ExecutionID*
+
+[[wizard]]
+== Wizard
```

Comment:
Is there anything you can add to the heading to be more descriptive?

Reply:
Oh! This is the task. Suggest moving this first and renaming it to be a user task and then group all the other topics in a reference section.